### PR TITLE
Fix disabled state

### PR DESCRIPTION
**Pull request by Kranex**
fixes the disabled state so form elements that are disabled, stay disabled after submission.

### CODE REVIEW

**Comment by Kranex** [comment]
@wouter-willems version number needs bumping before merging to master

File: projects/klippa/ngx-enhancy-forms/src/lib/form/form.component.ts [hunk 1]
```diff
@@ -50,20 +50,38 @@ export class FormComponent {
 	}
 	private disableInactiveFormControl(control: FormControl) {
 		if (this.activeControls.some((e) => e.formControl === control)) {
-			control.enable();
+			if (!control.disabled) {
```
Comment (wouter-willems) [inline projects/klippa/ngx-enhancy-forms/src/lib/form/form.component.ts:53]: I am wondering how much sense this makes now looking at it again.
If it is not disabled (therefore enabled), enabling again does nothing, right?

**Review by wouter-willems — COMMENTED** [review]
@Kranex reviewed

File: projects/klippa/ngx-enhancy-forms/src/lib/form/form.component.ts (hunk 1 above)
Comment (Kranex) [reply projects/klippa/ngx-enhancy-forms/src/lib/form/form.component.ts:53]: huh, yeah, was that if originally supposed to be:

```
if ( originallyEnabled? ) control.enable();
```

It definitely isn't being called now I and I think that it's not needed becasue the `trySubmit()` handles the re-enabling of states with the `setDisabledStatesForAllControls(originalDisabledStates)`

**Pull request merged**